source/js/donate-modal/donate-modal.jsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 import ReactGA from 'react-ga';
```

Comment:
Maybe as a followup -

Should we move `react-ga-proxy.js` up to `/source/js/` so main foundation site can use utilize it as well?

Reply:
we totally should.